### PR TITLE
feat(frontend): add contribution generic answer source

### LINKED ISSUE #3300: Ajout de la source de la fiche dans certaines contributions
# Problème

On a eu une réunion avec des personnes de Service-Public.fr, nous leur avons présenté les contributions notamment et on a remarqué une chose : lorsqu'on reprend entièrement la fiche Service-Public.fr on n'indique pas la source... :O
(on avait un peu honte de leur montrer cela...)

Exemple d'une contribution ici : https://cdtn-preprod-code-travail.dev2.fabrique.social.gouv.fr/contribution/a-quelles-indemnites-peut-pretendre-un-salarie-qui-part-a-la-retraite

# Solution 

Serait-il possible de rajouter la source de la fiche utilisée lorsqu'on utilise une fiche en entière (comme cela est traitée dans nos autre contenus) 

Exemple de visuel 
image

### PR DESCRIPTION
**Pull request by lionelB**
add sources to contributions when we embed an external content

fixes #3300

### CODE REVIEW

File: packages/code-du-travail-frontend/src/contributions/rehypeToReact.js [hunk 1]
```diff
@@ -24,18 +24,19 @@ const Hdn = (props) => (
...
-  <>
-    {raw && (
-      <StyledContent>
-        <FicheServicePublic data={JSON.parse(raw).children} />
-      </StyledContent>
-    )}
-  </>
-);
+const ContentSP = ({ raw }) => {
+  return (
```
Comment (m-maillot) [inline packages/code-du-travail-frontend/src/contributions/rehypeToReact.js:28]: Why did you add a return here? There is an issue with this syntax?

File: packages/code-du-travail-frontend/src/contributions/rehypeToReact.js [hunk 2]
```diff
@@ -24,18 +24,19 @@ const Hdn = (props) => (
...
+        </StyledContent>
+      )}
+    </>
+  );
+};
 
-const ContentMT = (props) => {
-  const { intro, sections } = props;
+const ContentMT = ({ intro, sections }) => {
   return (
```
Comment (m-maillot) [inline packages/code-du-travail-frontend/src/contributions/rehypeToReact.js:40]: The return can be omitted here. 
```js
const ContentMT = (props) => (
    <>
      {intro && ...
);
```

**Review by m-maillot — APPROVED** [review]
Minors feedbacks about the syntax of the code.

File: packages/code-du-travail-frontend/src/contributions/rehypeToReact.js (hunk 1 above)
Comment (lionelB) [reply packages/code-du-travail-frontend/src/contributions/rehypeToReact.js:28]: I have added a console log :) and I prefer this syntaxe since it's easier to add log statement or code before
